custom/icds_reports/dashboard_utils.py

```diff
@@ -13,6 +13,13 @@ def get_dashboard_template_context(domain, couch_user):
     context['all_user_location_id'] = list(couch_user.get_sql_locations(
         domain
     ).location_ids())
+    try:
+        if couch_user.is_commcare_user():
+            context['user_location_type'] = couch_user.sql_location.location_type_name
+        else:
+            context['user_location_type'] = couch_user.get_sql_location(domain).location_type_name
```

Comment:
Hey, i think you dont need to use the if condition because get_sql_location present in both classes.

Reply:
got it.resolved